docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,110 @@
+# Adding a New Metrics Provider for Placeholder Endpoint
+
+To create a provider for the placeholder endpoint, follow these steps:
+
+1. Fork the [Keptn repo](https://github.com/keptn/lifecycle-toolkit)
+
+2. **Define the Provider Type:** In the `metrics-operator/controllers/common/providers/common.go` file,
+ define the constant `KeptnPlaceholderProviderType` with the value `"placeholder"`.
+
+    ```go
+    const KeptnPlaceholderProviderType = "placeholder"
+    ```
+
+3. **Implement the Provider:** Create your own new folder inside the
+[metrics-operator/controllers/common/providers](https://github.com/keptn/lifecycle-toolkit/tree/main/metrics-operator/controllers/common/providers)
+ matching the new service name: placeholder and a new Go package for the placeholder provider.
+  This package should contain
+ a struct that implements the `KeptnSLIProvider` interface.
+  To fully implement the `KeptnSLIProvider` interface, it's necessary to implement the following functions.
+  `EvaluateQuery`(Fetches metric values from the provider)
+   - This function fetches metric values based on the provided
```

Comment:
point to the analysis type and metrics type so people know what to expect as input. 

ex This takes as input a KeptnMetric (link)

Reply:
Alright